app/renderer/components/Inspector/Commands.js

```diff
@@ -66,35 +65,54 @@ const Commands = (props) => {
     cancelPendingCommand();
   };
 
+  const generateCommandNotes = (notes) => {
+    let generatedNotes = [];
```

Comment:
This could be rewritten in more functional style:

```js
return notes
  .map((note) -> _.isArray(note) ? `${t(note[0])}: ${note[1]}` : t(note))
  .join('; ');
```